sqlite3/lib/src/wasm/file_system.dart

```diff
@@ -35,9 +37,16 @@ abstract class FileSystem {
   /// otherwise.
   void deleteFile(String path);
 
-  /// List all files stored in this file system.
+  /// Lists all files stored in this file system.
+  @Deprecated('Use files() instead')
```

Comment:
I think the deprecation notice should briefly explain why there's a new method now.